components/prism-core.js

```diff
@@ -236,21 +236,24 @@ var _ = {
 	highlightAllUnder: function(container, async, callback) {
 		var env = {
 			callback: callback,
+			container: container,
 			selector: 'code[class*="language-"], [class*="language-"] code, code[class*="lang-"], [class*="lang-"] code'
 		};
 
 		_.hooks.run('before-highlightall', env);
 
-		var elements = container.querySelectorAll(env.selector);
+		env.elements = Array.prototype.slice.apply(container.querySelectorAll(env.selector));
```

Comment:
Presumably, we're making `container` available on `env` so users can modify it via this hook.

```suggestion
		env.elements = Array.prototype.slice.apply(env.container.querySelectorAll(env.selector));
```